editors/vscode/package.json

```diff
@@ -801,6 +801,11 @@
         "category": "Typst",
         "icon": "$(file-pdf)"
       },
+      {
+        "command": "tinymist.copyAnsiHighlight",
+        "title": "Copy as Highlighted Code",
```

Comment:
The title is a little bit misleading. I remember vscode already has such feature.  If you copy from vscode and paste to word, you can see highlighted code. I believe this is the way that most people think of "copy as highlighted".

Maybe consider rename this to "copy as ansi highlighted code" or something?

Reply:
"Copy as Ansi Code"? (we need short title otherwise it will make the context menu's width very fat)